### PR TITLE
fixed homework2

### PR DESCRIPTION
**Pull request by MarkoAndrew93**

### CODE REVIEW

File: homeworks/andrew.markovskiy_MarkoAndrew93/Games/styles.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+a {
+  color: var(--color-1, #EF4934);
+  text-decoration-line: underline;
+}
+
+
+p {
+  color: var(--color-2, #221F1F);
```
Comment (MZabolotnev) [inline homeworks/andrew.markovskiy_MarkoAndrew93/Games/styles.css:13]: Why do you need a variable here?

File: homeworks/andrew.markovskiy_MarkoAndrew93/Games/styles.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+* {
+  box-sizing: border-box;
+}
+
+
+a {
+  color: var(--color-1, #EF4934);
```
Comment (MZabolotnev) [inline homeworks/andrew.markovskiy_MarkoAndrew93/Games/styles.css:7]: Why do you need a variable here?

**Review by MZabolotnev — CHANGES_REQUESTED** [review]

File: homeworks/andrew.markovskiy_MarkoAndrew93/Games/styles.css (hunk 1 above)
Comment (MarkoAndrew93) [reply homeworks/andrew.markovskiy_MarkoAndrew93/Games/styles.css:13]: the variable is taken from the layout

File: homeworks/andrew.markovskiy_MarkoAndrew93/Games/styles.css (hunk 2 above)
Comment (MarkoAndrew93) [reply homeworks/andrew.markovskiy_MarkoAndrew93/Games/styles.css:7]: the variable is taken from the layout

**Pull request closed without merging**